src/dodal/devices/xbpm_feedback.py

```diff
@@ -0,0 +1,27 @@
+from ophyd import Component, Device, EpicsSignal, EpicsSignalRO
+from ophyd.status import StableSubscriptionStatus, StatusBase
+
+
+class XBPMFeedback(Device):
+    """The XBPM feedback device is an IOC that moves the DCM, HFM and VFM to automatically
+    hold the beam into place, as measured by the XBPM sensor."""
+
+    # The time that the feedback needs to be high to be considered stable
```

Comment:
Good that we have a comment but can we expand it a little for people who aren't so familiar with the device?

Something along the lines of:

> We need to wait for `<something>` to go high, wait for `<some>` seconds, and then wait for it to go low again to ensure that `<some process>` has completed succesfully.